### PR TITLE
Replaced use of strings.ToLower comparison with EqualFold

### LINKED ISSUE #181: Replace use of strings.ToLower comparison with EqualFold
For example,
```
if strings.ToLower(jaeger.Spec.Strategy) == "streaming" {
```
should be replaced by
```
if strings.EqualFold(c.jaeger.Spec.Strategy, "streaming") {
```

### PR DESCRIPTION
**Pull request by tothandras**
Closes #181

Signed-off-by: András Tóth <andras.toth@banzaicloud.com>

### CODE REVIEW

**Comment by jpkrohling** [comment]
This change is [Reviewable](https://reviewable.io/reviews/jaegertracing/jaeger-operator/214)

**Comment by tothandras** [comment]
oh, my regex wasn't the best, 1 sec

File: pkg/cronjob/spark_dependencies.go [hunk 1]
```diff
@@ -17,7 +17,13 @@ import (
 var supportedStorageTypes = map[string]bool{"elasticsearch": true, "cassandra": true}
 
 func SupportedStorage(storage string) bool {
-	return supportedStorageTypes[strings.ToLower(storage)]
```
Comment (tothandras) [inline pkg/cronjob/spark_dependencies.go]: @jpkrohling `codecov` is not happy about this change.

**Comment by tothandras** [comment]
___
*[pkg/cronjob/spark_dependencies.go, line 20 at r3](https://reviewable.io/reviews/jaegertracing/jaeger-operator/214#-LZF7EMcBkhCLNbGcAM_:-LZF8yaJ-3L0ghwG8vSA:b-2fbi2u) ([raw file](https://github.com/jaegertracing/jaeger-operator/blob/8bfea6badeeb0bd4e6f97c5f77f0fcfa84db1cc8/pkg/cronjob/spark_dependencies.go#L20)):*
<details><summary><i>Previously, jpkrohling (Juraci Paixão Kröhling) wrote…</i></summary><blockquote>

Cool, I guess you found one place where it should indeed be `ToLower` ;) 
</blockquote></details>

okay, reverted

**Comment by jpkrohling** [comment]
Thanks!

**Pull request merged**